### ISSUE TITLE
Create custom site

### ISSUE THREAD

**Issue by brettfarley** [brettfarley/personal#1]
http://indiegroundthemes.com/journey/version3/

![screen shot 2016-06-06 at 5 50 18 pm](https://cloud.githubusercontent.com/assets/13319038/15895711/5dc5ef44-2d52-11e6-882f-95aafc2068c2.jpg)

**Comment by supportast** [comment]
started working on local .Will put on domain and show you the progress .

**Comment by brettfarley** [comment]
ok thanks

**Comment by supportast** [comment]
styling is pending 
http://vsstaging.net/journey/

**Comment by brettfarley** [comment]
very cool

**Comment by supportast** [comment]
done

**Comment by brettfarley** [comment]
Please also match the top portion where the header (where the logo is) will be a slider. The middle section will be my avatar, info, social profile buttons.
![screen shot 2016-07-09 at 9 39 28 am](https://cloud.githubusercontent.com/assets/13319038/16708429/27f0b8fc-45b9-11e6-9ea8-202c8a58df41.jpg)

**Comment by supportast** [comment]
worked on it . styling is pending

**Comment by supportast** [comment]
Few styling issues pending .

**Comment by supportast** [comment]
Please check now and share your feedback . what else is required to be done

**Comment by brettfarley** [comment]
Ok, here are some tweaks. 
1. Let's make the masthead a slider, similar to the one we have at OKGOP.
2. The body area with the avatar and biographic text needs to be shifted to the left because it looks off-center.
3. Let's put the social icons on the right hand side in the menu bar.
4. Let's make the pics at the bottom fixed instead of as a slider. We basically just need 5 images with headline text overlayed.
5. Can we make the bio information toggle like the FAQ at OKGOP? So maybe the first paragraph is viewable and then additional text will expand when toggled?

**Comment by supportast** [comment]
1. Let's make the masthead a slider, similar to the one we have at OKGOP. - Do we need to make header a slider . please clarify 

working on others ..

**Comment by brettfarley** [comment]
Yes. Instead of the current fixed header with the logo, I want a slider.

**Comment by supportast** [comment]
1. Let's make the masthead a slider, similar to the one we have at OKGOP. - Done , please check the size and let me know your feedback 
2. The body area with the avatar and biographic text needs to be shifted to the left because it looks off-center. - done 
3. Let's put the social icons on the right hand side in the menu bar. - done 
4. Let's make the pics at the bottom fixed instead of as a slider. We basically just need 5 images with headline text overlayed.
5. Can we make the bio information toggle like the FAQ at OKGOP? So maybe the first paragraph is viewable and then additional text will expand when toggled? - done , please check and let me know the content and icon to be changed or not .

**Comment by brettfarley** [comment]
1. Please also delete the social icons just below the bio toggle button.
2. Please edit the size of the pics so they all fit on the screen.
3. Please change "expand" to "Read more..."

**Comment by supportast** [comment]
done ... Do I need to reduce the gap between the text and read more .

**Comment by brettfarley** [comment]
The gap is fine. But now the slider isn't running to the edge of the screen. There are white lines are either side. The slider should expand/contract with the browser window just like it does on the OKGOP site. Also the images at the bottom should similarly run edge to edge and expand with the browser window. On Safari, the images run off the screen and on Chrome they aren't wide enough. See screenshots.

![screen shot 2016-08-11 at 8 01 31 am](https://cloud.githubusercontent.com/assets/13319038/17589403/425ced32-5f9a-11e6-881e-856f106afa43.jpg)
![screen shot 2016-08-11 at 8 03 25 am](https://cloud.githubusercontent.com/assets/13319038/17589405/42725046-5f9a-11e6-95ff-473d105db335.jpg)
![screen shot 2016-08-11 at 8 04 27 am](https://cloud.githubusercontent.com/assets/13319038/17589404/4270ad7c-5f9a-11e6-9380-2e93952bc317.jpg)

**Comment by brettfarley** [comment]
Also, the social icons in the menu bar don't seem to be individual icons and they aren't linking anywhere. Shweta, I need you to take the time to do correct work the first time so I don't have to ask for corrections. The time that I'm paying for shouldn't include additional time because things aren't done correctly.

**Comment by brettfarley** [comment]
Please install this theme at personal.vsstaging.net.
[themeforest-12234742-journey-personal-wordpress-blog-theme.zip](https://github.com/brettfarley/personal/files/413737/themeforest-12234742-journey-personal-wordpress-blog-theme.zip)

**Comment by supportast** [comment]
theme installed and work in progress.. you can check and share your feedback ..
